python/altrios/altrios_pyo3.pyi

```diff
@@ -983,6 +983,8 @@ class Heading(SerdeAPI):
     @classmethod
     def default(cls) -> Self: ...
 
+class SpeedSet(SerdeAPI): ...
+    # TODO: finish fleshing this out
```

Comment:
@sakhtar312 , if you have bandwidth to  take a stab at this before merging, that'd be helpful.  Otherwise, we can take care of it later.